README.md

```diff
@@ -47,8 +47,10 @@ gPXE images are unsupported due to lack of initrd support.
 ## Configuration
 
 With all image types, hosts have to be registered to Foreman prior to booting
-the image.  Hosts will be identified by their MAC or IP address to provide
+the image by hand.  Hosts will be identified by their MAC or IP address to provide
 the correct provisioning template if the host is in build mode.
+On VMWare compute resources it is possible to create a new host with a cd-drive
```

Comment:
Can you expand on this?  Split it into a new paragraph and state that you use it by selecting the Bootdisk Based provisioning method under the OS tab when creating the host, or by setting `provision_method: "bootdisk"` using the API.